src/FakeItEasy/ReturnValueConfigurationExtensions.cs

```diff
@@ -122,7 +122,7 @@ public static void ReturnsNextFromSequence<T>(this IReturnValueConfiguration<Tas
         /// </summary>
         /// <param name="configuration">The call configuration to extend.</param>
         /// <returns>A configuration object.</returns>
-        public static IAfterCallConfiguredWithOutAndRefParametersConfiguration<IReturnValueConfiguration<Task>> DoesNothing(this IReturnValueConfiguration<Task> configuration)
+        public static IAfterCallConfiguredConfiguration<IReturnValueConfiguration<Task>> DoesNothing(this IReturnValueConfiguration<Task> configuration)

```

Comment:
The return type was wrong (not the same as in `IDoNothingConfiguration`)
I made a fixup commit so that you can see the difference